lib/diagnostic.js

```diff
@@ -39,24 +52,40 @@ let owner = null;
 // Methods: vocalize
 
 // Replace null with your answer
-let pet = null;
+
+const useVocalization = function () {
+  return vocalization
+}
+
+let pet = {
+  name: 'Buddy',
+  species: 'Dog',
+  breed: 'Golden Retriever',
+  vocalization: 'bark',
+  vocalize: useVocalization
+}
 
 // 4. Create a constructor function for pet objects, as described above. The
 // constructor function should be responsible for setting the values of `name`,
 // `species`, `breed`, and `vocalization`.
 
-const Pet = function () {
-  /* Your answer here */
-};
+const Pet = function Pet (name, species, breed, vocalization) {
+  this.name = name
+  this.alias = alias
+  this.breed = breed
+  this.vocalization = vocalization
+}
 
 // 5. Attach a `vocalize` method to the Pet's prototype.  The `vocalize` method
 // should return the pet's vocalization attribute.
 
-/* your answer here */
+const useVocalization = function () {
+  return vocalization
+}
```

Comment:
You want to attach this to the prototype, not the object itself. Something like this: `Pet.prototype.useVocalization = function(){return this.vocalization}`. Let me know if you need some more clarification on how to use prototypes.